cmd/loadtest/app.go

```diff
@@ -18,6 +18,28 @@ import (
 	"golang.org/x/time/rate"
 )
 
+
+type BigIntValue struct {
```

Comment:
```suggestion
type bigIntValue struct {
```

I think it would be cleaner to follow name casing convention used here like `blockSummary`, `hexwordReader`, etc..
Same for all other `BigIntValue` occurrences. wdyt?